developersociety/django-template
# Switch Wagtail Font Awesome package

To test:

```
mktmpenv --python=python3.10
cookiecutter --no-input --checkout font-awesome-svg gh:developersociety/django-template wagtail=y
cd projectname
nvm use
make npm-install pip-install-local
dropdb --if-exists projectname_django
createdb projectname_django
./manage.py migrate
make django-dev-createsuperuser
./manage.py runserver
```

Check: http://127.0.0.1:8000/admin/pages/3/edit/

- Check all the icons in the menu bar - settings social share should be visible
- Check the text block icon - should also now be visible
